test/e2e/esm-externals/next.config.js

```diff
@@ -1,21 +1,9 @@
 module.exports = {
   experimental: {
-    turbo: {
-      resolveAlias: {
-        'preact/compat': 'react',
-      },
-    },
```

Comment:
That's testing that externals still work after aliasing. Better keep that test case.